app/router.js

```diff
@@ -1,13 +1,21 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-const Router = Ember.Router.extend({
+const { Router } = Ember;
+
+const ApplicationRouter = Router.extend({
   location: config.locationType,
   rootURL: config.rootURL
 });
 
-Router.map(function() {
+ApplicationRouter.map(function() {
   this.route('users-list');
+  this.route('products', function() {
+    this.route('product', { path: '/:product_id' });
+    this.route('edit', { path: '/:product_id/edit' });
+    this.route('new', { path: '/new' });
+    this.route('destroy', { path: '/:product_id/destroy' });
```

Comment:
do we need (and do we use) this route?

I think we don't need it, as we delete directly form the products listing, pls check it.